presto-main/src/main/java/io/prestosql/connector/ConnectorManager.java

```diff
@@ -187,12 +187,12 @@ private synchronized CatalogName createConnection(String catalogName, ConnectorF
         CatalogName catalog = new CatalogName(catalogName);
         checkState(!connectors.containsKey(catalog), "Catalog '%s' already exists", catalog);
 
-        addCatalogConnector(catalog, connectorFactory, properties);
+        createCatalog(catalog, connectorFactory, properties);
 
         return catalog;
     }
 
-    private synchronized void addCatalogConnector(CatalogName catalogName, ConnectorFactory factory, Map<String, String> properties)
+    private synchronized void createCatalog(CatalogName catalogName, ConnectorFactory factory, Map<String, String> properties)
```

Comment:
`createCatalogConnector` ?

(it wasn't an overload before)

Reply:
CatalogConnector? This sounds like a butter made from milk.